appendix-basic65.tex

```diff
@@ -3457,9 +3463,19 @@ \subsection{EDMA}
                  target [{\bf,} sub{\bf,} mod]
 \item [Usage:]
    {\bf EDMA} ("Extended Direct Memory Access") is the fastest method
-   to manipulate memory areas using the DMA controller.
+   to manipulate memory areas using the DMA controller. Please refer to
+\ifdefined\printmanual
+the {\bf MEGA65 Book}
+\else
+\bookvref{cha:dmagic}
+\fi
+   for the powerful possibilities with EDMA.
```

Comment:
Oh, this kind of reference isn't limited to the BASIC or the user-guide PDF because both of them are present in the big book aswell. Let me assure you that I tested all references that were `??` before. They all are right.

For the phrasing I'm editing right now. Await the next commit of mine.